packages/gluestick-env-expose-ast/package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "test": "jest",
     "test-coverage": "jest --coverage",
-    "lint": "eslint ./",
+    "lint": "eslint ./*.js ./__tests__/**/*.js",
```

Comment:
`.eslintignore` should be in root folder, next to `.eslint` because `coverage` folder will be in every package